google/cloud/internal/backoff_policy.h

```diff
@@ -58,9 +58,14 @@ class BackoffPolicy {
   virtual std::unique_ptr<BackoffPolicy> clone() const = 0;
 
   /**
-   * Return the delay after an RPC operation has completed.
+   * Handle an operation completion.
    *
-   * @return how long to wait before trying the operation again.
+   * This function is typically called when an operation has failed (if it had
```

Comment:
Nit: This is a big run-on sentence now.  Switch out all the commas for periods, please :)

Also, the doc summary is much nicer now. Thanks!

Reply:
Fixed.